README.md

```diff
@@ -1,4 +1,8 @@
-![icon](https://cl.ly/0Z2P3Y3D060K/rfi.svg)
+![icon-features](http://f.cl.ly/items/2X1V2O0U2z453G0O292J/react-file-icons-default.svg)
```

Comment:
I'm getting some weird rendering issues: 
![screen shot 2018-02-14 at 8 24 13 am](https://user-images.githubusercontent.com/957488/36215310-96ea9312-1160-11e8-90a2-df56009c5ee8.png)

Seeing the height of these it might be better to create an "Examples" section above "Props" and include them there?

Reply:
Resolved